INSTALL.md

```diff
@@ -23,19 +23,10 @@ conda activate maskrcnn_benchmark
 # this installs the right pip and dependencies for the fresh python
 conda install ipython
 
-# maskrcnn_benchmark and coco api dependencies
-pip install -r requirements.txt
-
-# follow PyTorch installation in https://pytorch.org/get-started/locally/
```

Comment:
Can you keep this comment? This is very helpful for beginners, specially if they want to change the CUDA version

Reply:
I also vote for keeping this part.  Personally, I like to compile PyTorch by myself instead of installing from conda/pip.

Reply:
Makes sense indeed. Just put it back.